content/en/observability_pipelines/guide/sensitive_data_scanner_transform.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Sensitive Data Scanner Transform
+kind: documentation
+disable_toc: false
+private: true
+is_beta: true
+further_reading:
+- link: "/observability_pipelines/setup/"
+  tag: "Documentation"
+  text: "Set up Observability Pipelines"
+- link: "/observability_pipelines/working_with_data/"
+  tag: "Documentation"
+  text: "Working with data in Observability Pipelines"
+---
+
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSfnNnV823zAgOCowCYuXJE5cDtRqIipKsYcNpaOo1LKpGfppA/viewform" btn_hidden="false" header="Request Access!">}}
+The <code>sensitive_data_scanner</code> transform is in private beta.
+{{< /callout >}}
+
+## Overview
+
+Sensitive data, such as credit card numbers, bank routing numbers, and API keys, are often exposed unintentionally in your logs, which can expose your organization to financial and privacy risks. Use the Observability Pipelines `sensitive_data_scanner` transform to identify, tag, and optionally redact or hash sensitive information before routing data to different destinations. You can use out-of-the-box scanning rules to detect common patterns such as email addresses, credit card numbers, API keys, authorization tokens, and more. Or, create custom scanning rules using regex patterns to match sensitive information.
+
+## Set up the `sensitive_data_scanner` transform
```

Comment:
Is there a way to break this up into sub sections? Like Add a component, Add a rule, Define an action, etc?

Reply:
I'll look into this in the next iteration, thanks!